vllm/engine/async_llm_engine.py

```diff
@@ -435,6 +439,7 @@ async def add_request_async(
             lora_request=lora_request,
             prompt_adapter_request=prompt_adapter_request,
             trace_headers=trace_headers,
+            priority=priority,
```

Comment:
i don't think we are passing `policy="priority"` anywhere, shall we add it to [here](https://github.com/vllm-project/vllm/blob/3b00b9c26c91e9f9ada12975b613555698054e39/vllm/engine/arg_utils.py#L1009) so that user can actually leverage this. Otherwise, how are folks supposed to use this feature? @njhill

Reply:
I think currently the only way of actually using the feature is to manually change the policy after creating the engine.

Reply:
yes, but shall we allow folks to specify through args? Like other param in the scheduler_config? I feel that's better for wider adoption for this feature.

Reply:
Sorry, I misread your comment. Yes I think this needs to go in the `EngineArgs` or some similar place. However, I think that should be a separate PR.

Reply:
Yeah it's in SchedulerConfig but still needs to be wired in the EngineArgs to be able to enable it externally. Agree that could be done in a separate PR